test/functional/fixtures/api/es-next/assertions/testcafe-fixtures/assertions-test.js

```diff
@@ -155,3 +155,7 @@ test('ClientFunction result assertion', async t => {
 
     await t.expect(getSomeVar()).eql(2);
 });
+
+test('Assertion without method call', async t => {
+    await t.expect();
```

Comment:
It's correct. The `expect` method returns `Assertion` object, which does not have `click` method

Reply:
Yep, I know how it works. But at first this behavior seemed to me odd.
I've seen initial issue, now I see we are trying to avoid false-positive test passing.
So, ok.